src/config.ts

```diff
@@ -0,0 +1,6 @@
+const config = {
+    apiUrl: "https://example.com/api",
+    appVersion: "1.0.0",
+  };
```

Comment:
How do you handle those environment variables?
Will it be part of a next PR that will inject an `env.js` script in the `index.html` file?